web/models/pair.ex

```diff
@@ -4,16 +4,16 @@ defmodule Pairmotron.Pair do
   schema "pairs" do
     field :year, :integer
     field :week, :integer
-    many_to_many :users, Pairmotron.User, join_through: "users_pairs"
+    many_to_many :users, Pairmotron.User, join_through: Pairmotron.UserPair
```

Comment:
Had to make this explicit so that the ex machina factory wouldn't encounter a Postgrex error because timestamps were not inserted.